docs/samples/contact-center/app.js

```
@@ -1166,14 +1196,181 @@ function wrapupCall() {
   wrapupElm.disabled = true;
   const wrapupReason = wrapupCodesDropdownElm.options[wrapupCodesDropdownElm.selectedIndex].text;
   const auxCodeId = wrapupCodesDropdownElm.options[wrapupCodesDropdownElm.selectedIndex].value;
-  task.wrapup({wrapUpReason: wrapupReason, auxCodeId: auxCodeId}).then(() => {
+  currentTask.wrapup({wrapUpReason: wrapupReason, auxCodeId: auxCodeId}).then(() => {
     console.info('Call wrapped up successfully');
     holdResumeElm.innerText = 'Hold';
     holdResumeElm.disabled = true;
     endElm.disabled = true;
     wrapupCodesDropdownElm.disabled = true;
+    updateTaskList();
   }).catch((error) => {
     console.error('Failed to wrap up the call', error);
     wrapupElm.disabled = false;
   });
-}
\ No newline at end of file
+}
+
+function acceptTask(task) {
+  const taskId = task?.data?.interactionId;
+  if (!taskId) return;
```

We can show some message or throw an error if taskId is not found

---

addressed